instrumentation/openai-client-instrumentation/instrumentation-1.1/src/main/java/co/elastic/otel/openai/v1_1/OpenAiOkHttpClientBuilderInstrumentation.java

```diff
@@ -51,7 +51,7 @@ public static class AdviceClass {
     public static OpenAIClient onExit(
         @Advice.Return OpenAIClient result,
         @Advice.FieldValue("clientOptions") ClientOptions.Builder clientOptions) {
-      return InstrumentedOpenAiClient.wrap(result).baseUrl(clientOptions.baseUrl()).build();
+      return InstrumentedOpenAiClient.wrap(result).baseUrl(clientOptions.build().baseUrl()).build();
```

Comment:
[for reviewer] this means the builder `.build()` method will be executed twice, but a quick look at the implementation indicates that there isn't any non-idempotent operation in it, which means it should be fine.